fuel-vm/src/interpreter/blockchain/other_tests.rs

```diff
@@ -251,26 +251,35 @@ fn test_code_root() {
         },
     };
     let mut pc = 4;
+    let is = 0;
+    let mut cgas = 0;
+    let mut ggas = 0;
```

Comment:
It would be nice to check that `cgas` and `ggas` are udpated=)

Reply:
I refactored the CROO test into `croo_tests.rs` and broke it up into individual tests. Each test now checks the `cgas` and `ggas` explicitly, for both success and failure cases.